setup.py

```diff
@@ -6,34 +6,22 @@
     long_description = fh.read()
 
 dependencies = [
-    "chia-blockchain==1.8.1",
-    "packaging",
+    "chia-blockchain==2.0.0",
 ]
 
 dev_dependencies = [
-    "build",
-    "coverage",
-    "pre-commit",
-    "pylint",
-    "pytest",
-    "pytest-asyncio>=0.18.1",  # require attribute 'fixture'
-    "isort",
-    "flake8",
-    "mypy",
-    "black==21.12b0",
-    "aiohttp_cors",  # For blackd
-    "ipython",  # For asyncio debugging
-    "pyinstaller==5.0",
-    "pytest",
-    "pytest-asyncio",
-    "pytest-env",
-    "types-aiofiles",
-    "types-click",
-    "types-cryptography",
-    "types-pkg_resources",
-    "types-pyyaml",
-    "types-setuptools",
-    "types-docutils",
+    "pre-commit==3.3.3",
```

Comment:
Do we both have dependabot enabled here and also keep up with the PRs it generates to update versions?